### PR TITLE
Dropdowns: Export full Tippy dropdown

### PR DESCRIPTION
**Pull request by zdmitchell**
Originally, we copied over `LangaugeDropdown` and `StoreDropdown` into this repo, but Inkofpixel wants the entire Tippy implementation, since they need something basically identical to our desktop versions. This is essentially a straight-copy paste of the essential files. I also updated the example page to demonstrate how to use them.

## QA
Ensure that the dropdowns on the example page look the same as the ones on master. Additionally, ensure that everything is exported correctly using the following steps:
1. Create a new Create React App using `npx create-react-app ppc-test`
2. Run `npm build` then `npm pack` in this repo's directory.
3. Copy over `ifixit-product-page-components-1.1.9.tgz`, `stores.json`, and `languages.json` from this repos directory to your new directory (`stores.json` and `languages.json` should go under `src`).
4. In the new directory, run `npm install ifixit-product-page-components-1.1.9.tgz`.
5. Run `npm install styled-components`.
5. Copy the following code to `App.js` and ensure everything compiles/renders correctly.

<details>
<summary>App.js</summary>

```
import {
   LanguageDropdownContainer,
   StoreDropdown,
   Dropdown,
   StoreFlag,
} from "@ifixit/product-page-components";
import languages from "./languages.json";
import stores from "./stores.json";
import styled from "styled-components";

const AppContainer = styled.div`
   display: flex;
   justify-content: center;
   align-items: center;
   height: 1000px;

   a {
      text-decoration: none;
   }

   .dropdown-and-button-container {
      margin: 0 10px;
   }
`;

const StoreSelector = ({ stores, onClickStore, ...tippyOptions }) => {
   const currentStore = stores.find(store => store.isDisplayStore);

   return (
      <Dropdown
         icon={<StoreFlag storeCode={currentStore.storeCode} />}
         {...tippyOptions}
      >
         <StoreDropdown
            currentStore={currentStore}
            stores={stores}
            onClickStore={onClickStore}
         />
      </Dropdown>
   );
};

function App() {
   return (
      <AppContainer>
         <StoreSelector
            stores={stores}
            onClickStore={store => console.log(store)}
            placement="top-end"
         />
         <LanguageDropdownContainer
            languages={languages}
            translationPreferencesUrl="https://www.cominor.com/Login"
            machineTranslationRequested={false}
            placement="top-end"
         />
      </AppContainer>
   );
}

export default App;
```
</details>

Connects https://github.com/iFixit/ifixit/issues/35956

### CODE REVIEW

**Comment by zdmitchell** [comment]
dev_block :guitar: on https://github.com/iFixit/ifixit/pull/36182 being merged first.

**Comment by zdmitchell** [comment]
un_dev_block :guitar:

**Comment by ardelato** [comment]
QA 👍 

The components are still working and look similar to masters. 
Was able to import the package into a new react app and compile the page without any issues. 

deploy_block❓ is there a font-family that is being used with this repo? It seems the font-family being applied to the components imported as an npm package is different than the live site and this repo's demo page

| Live Site	| Demo Page	| New React App Page  |
|:---:|:---:|:---:|
|Live Site Styling|Font Styling	| Font Styling Import|

## Steps Taken:
1. Mainly followed the steps from this guide https://overlord.dozuki.com/Wiki/QA_Testing_the_Repo_%22product-page-components%22
2. Added the modifications from the QA steps stated in this Pull's body.

**Comment by masonmcelvain** [comment]
CR 👍 (looked at this earlier but forgot to cr it 😅 )

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "@core-ds/primitives": "^1.0.0",
     "@ifixit/localize": "^1.0.0",
     "@ifixit/toolbox": "^1.1.0",
+    "@tippy.js/react": "^3.1.1",
```
Comment (danielbeardsley) [inline package.json:14]: Now this is incompatible with the version in our main repo. i.e. we'll have to have two copies of tippy. We should probably downgrade this one or upgrade the other.

**Comment by danielbeardsley** [comment]
CR 👍

File: package.json (hunk 1 above)
Comment (danielbeardsley) [reply package.json:14]: FYI, I don't mean it won't work, just that npm will now have to include both versions of tippy.

Comment (zdmitchell) [reply package.json]: Fixed in https://github.com/iFixit/product-page-components/pull/34/commits/571e2997b03123cfd41cd945bdef4750c2d637c4.

**Comment by zdmitchell** [comment]
> It seems the font-family being applied to the components imported as an npm package is different than the live site and this repo's demo page

I made the demo page use the same font-family as the live site in https://github.com/iFixit/product-page-components/pull/34/commits/2a8e97a203d122a1a50cc9889a52e411b029e1f7. If you test using this in a new, fresh repo, you'll have to give the `body` element the same style: `font-family: Lato, Arial, sans-serif;`

un_deploy_block :guitar:

**Comment by masonmcelvain** [comment]
CR :+1:

**Comment by danielbeardsley** [comment]
CR 👍

**Comment by ardelato** [comment]
QA 👍 

The dropdowns still behave similar to the live site and with the additional **font-family** property added they now look similar to the live site as well. 



## Steps Taken:
1. Followed the steps from https://github.com/iFixit/product-page-components/pull/34#issuecomment-774366041
2. However, before running `npm install <file.tgz>` I had to downgrade my react version in the `package.json` of the new react app
3. Basically I just modified the react version number in the `package.json` file from **17.0.1** to **16.0.0** in order to properly install the `<file.tgz>` package.

**Pull request merged**